packages/client/components/promptResponse/tiptapConfig.ts

```diff
@@ -98,6 +98,7 @@ export const createEditorExtensions = (
     openOnClick: isReadOnly
   }),
   Placeholder.configure({
+    showOnlyWhenEditable: false,
```

Comment:
Not setting that means we won't show the placeholder when tip tap editor isn't in editable state ie. when we end the meeting, we still want to show a placeholder `No response`